samples/nrf9160/lwm2m_client/prj.conf

```diff
@@ -121,12 +121,3 @@ CONFIG_EVENT_MANAGER=y
 # Date-Time library
 CONFIG_DATE_TIME=y
 CONFIG_DATE_TIME_UPDATE_INTERVAL_SECONDS=86400
-
-CONFIG_LWM2M_VERSION_1_1=y
-CONFIG_LWM2M_PORTFOLIO_OBJ_SUPPORT=y
-CONFIG_BASE64=y
-CONFIG_LWM2M_RW_SENML_JSON_SUPPORT=y
-CONFIG_LWM2M_RW_JSON_SUPPORT=y
-CONFIG_LWM2M_SECONDS_TO_UPDATE_EARLY=8
-CONFIG_LWM2M_COAP_MAX_MSG_SIZE=1024
-CONFIG_LWM2M_COMPOSITE_PATH_LIST_SIZE=10
```

Comment:
Why do you touch the defaults in this commit?

Reply:
Default was 1.1 conformance if you want keep it I can leave those.